pom.xml

```diff
@@ -2905,6 +2907,8 @@
                 <param>com.puppycrawl.tools.checkstyle.grammar.javadoc.JavadocParseTreeTest</param>
                 <!-- this test is required for Checker -->
                 <param>com.puppycrawl.tools.checkstyle.filefilters.BeforeExecutionExclusionFileFilterTest</param>
+                <!-- this test is required for BeanPropertyType -->
+                <param>com.puppycrawl.tools.checkstyle.internal.XdocsPagesTest</param>
```

Comment:
`XdocPagesTest` is a big test (takes a few seconds to run) to be added for this. Why not just create a test for the class specifically?

Reply:
done

Reply:
@romani doesn't like synthetic tests, but in this case it makes sense: we need to at least assert that all types are used.